syntax = "proto3";

//TODO: SE와 통신하기 위한 임시 protobuf 입니다.수정 될 것.
package grpc_se;

service  GruutSeService {
    rpc transaction(GrpcMsgTX) returns (TxStatus) {}
}
message GrpcMsgTX {
    bytes message = 1;
}
message TxStatus {
    bool message = 1;
}